packages/node-http-handler/src/node-http-handler.spec.ts

```diff
@@ -18,12 +18,12 @@ import {
 describe("NodeHttpHandler", () => {
   describe("constructor", () => {
     it("can set httpAgent and httpsAgent", () => {
-      let maxSockets = Math.round(Math.random() * 50);
+      let maxSockets = Math.round(Math.random() * 50) + 1;
```

Comment:
As `Math.random()` is inclusive of `0`, using `Math.ceil()` wouldn't be sufficient here.

Reply:
@trivikr I added this comment before your review in the hope you *wouldn't* suggest `Math.ceil()` 😆

Reply:
Haha! I just read the comment after posting my comments 😜